src/views/folderNode.ts

```diff
@@ -33,6 +33,6 @@ export class FolderNode extends DataNode {
     }
 
     protected get iconPath(): ThemeIcon {
-        return ThemeIcon.Folder;
+        return ExplorerNode.resolveIconPath("packagefolder");
```

Comment:
Should use the ThemeIcon.Folder which is the VSCode provided. To keep consistent with UI. THe packagefolder will have